Makefile

```diff
@@ -256,14 +256,13 @@ docker-build-access-debug:
 	docker build -f cmd/Dockerfile  --build-arg TARGET=access  --build-arg COMMIT=$(COMMIT) --build-arg VERSION=$(IMAGE_TAG) --build-arg GOARCH=$(GOARCH) --target debug \
 		-t "$(CONTAINER_REGISTRY)/access-debug:latest" -t "$(CONTAINER_REGISTRY)/access-debug:$(SHORT_COMMIT)" -t "$(CONTAINER_REGISTRY)/access-debug:$(IMAGE_TAG)" .
 
-# Observer is currently simply access node, this target is added for compatibility with deployment pipeline
-# Once proper observer is separated in the code, we should just need to change TARGET parameter below
+# Observer is currently a modified access node, this target is added for compatibility with deployment pipeline
+# It points to cmd/observer
```

Comment:
Let's remove these comments entirely, an observer and Access Node are entirely separate at this point.

Reply:
@smnzhu Thank you for the review. I am addressing these now.